### PR TITLE
Fix wrong words and ordering for staff member form

### PR DESCRIPTION
**Pull request by kamilpastuszka**
I want to merge this to fix wrong words and ordering for staff member form, as described in the ticket.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2147 Wrong Description/Title/Ordering in Staff Member Form](https://app.clickup.com/t/2549495/SALEOR-2147)

File: src/staff/components/UserStatus/UserStatus.tsx [hunk 1]
```diff
@@ -21,17 +21,18 @@ const AppStatus: React.FC<AppStatusProps> = ({
 }) => {
   const intl = useIntl();
 
+  //
```
Comment (krzysztofwolski) [inline src/staff/components/UserStatus/UserStatus.tsx:24]: :police_car:

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: src/staff/components/StaffDetailsPage/StaffDetailsPage.tsx [hunk 2]
```diff
@@ -143,6 +143,16 @@ const StaffDetailsPage: React.FC<StaffDetailsPageProps> = ({
                 )}
                 {canEditStatus && (
                   <>
+                    <UserStatus
+                      data={formData}
+                      disabled={disabled}
+                      label={intl.formatMessage({
+                        defaultMessage: "User is active",
```
Comment (bmigirl) [inline src/staff/components/StaffDetailsPage/StaffDetailsPage.tsx:150]: Move to local messages file

File: src/staff/components/UserStatus/UserStatus.tsx [hunk 3]
```diff
@@ -25,13 +25,13 @@ const AppStatus: React.FC<AppStatusProps> = ({
     <Card>
       <CardTitle
         title={intl.formatMessage({
-          defaultMessage: "App Status",
+          defaultMessage: "User Status",
```
Comment (bmigirl) [inline src/staff/components/UserStatus/UserStatus.tsx:28]: Same as above

File: src/staff/components/UserStatus/UserStatus.tsx [hunk 4]
```diff
@@ -25,13 +25,13 @@ const AppStatus: React.FC<AppStatusProps> = ({
...
           description: "section header"
         })}
       />
       <CardContent>
         <Typography variant="body2">
-          <FormattedMessage defaultMessage="If you want to disable this App please uncheck the box below." />
+          <FormattedMessage defaultMessage="If you want to disable this User please uncheck the box below." />
```
Comment (bmigirl) [inline src/staff/components/UserStatus/UserStatus.tsx:34]: Same as above

**Review by bmigirl — CHANGES_REQUESTED** [review]

**Review by krzysztofwolski — APPROVED** [review]

File: src/intl.ts [hunk 5]
```diff
@@ -112,6 +112,18 @@ export const commonMessages = defineMessages({
     defaultMessage: "Upload image",
     description: "button"
   },
+  userStatus: {
```
Comment (bmigirl) [inline src/intl.ts:115]: These messages should be in local messages file, not it the main / common messages.

**Review by bmigirl — CHANGES_REQUESTED** [review]

**Review by bmigirl — APPROVED** [review]

**Pull request merged**